src/wallet/KintoWallet.sol

```diff
@@ -282,8 +282,8 @@ contract KintoWallet is Initializable, BaseAccount, TokenCallbackHandler, IKinto
         // if app key is not set or signature is not valid, verify signer policy
         if (
             (
-                signerPolicy == SINGLE_SIGNER && owners.length == 1
```

Comment:
I dont event know if this is needed because if he has 1 or 2 signers the policy should be signers minus 1 not single signer so maybe we dont need to change anything

Reply:
not sure i follow you... AFAIK we have 3 policies:

1. Single signer: requires one signature (`owners.length` could be any length) -- _signers could be any of the owners_ ❌ 

2. Signer minus one: requires `owners.length - 1` signatures -- _signers could be any of the owners ✅, signatures can be in any order_ ✅

3. All signers: requires `owners.length` signatures -- _signers could be any of the owners ✅ (signers are, in fact, all the owners), signatures can be in any order_ ✅

is this correct?

the condition that is not being met is the 1st one and this is what is being fixed on the PR

Reply:
we talked via TG. Single signer is just meant for one owner